apps/src/templates/instructions/utils.js

```diff
@@ -0,0 +1,32 @@
+import ReactDOM from 'react-dom';
+
+/**
+ * @param {ReactComponent} component
+ * @param {boolean} includeMargin
+ * @returns {number} The current computed height in pixels of the specified component,
+ * including padding, border, and (optionally) margins.
+ */
+export function getOuterHeight(component, includeMargin=false) {
+  return $(ReactDOM.findDOMNode(component)).outerHeight(includeMargin);
+}
+
+/**
+ * Manually scrolls the specified element by the specified delta
+ * @param {Element} element The element to scroll
+ * @param {number} deltaY The distance (positive or negative) in pixels
+ *        to scroll by
+ * @param {number} animate If specified, the animation time in ms
+ */
+export function scrollBy(element, deltaY, animate=200) {
+  let newScrollTop = element.scrollTop + deltaY;
+  if (animate) {
+    let $elem = $(element);
+    if (!$elem.is(':animated')) {
+      $elem.animate({
+        scrollTop: newScrollTop
+      }, animate);
+    }
+  } else {
+    element.scrollTop = newScrollTop;
+  }
+}
```

Comment:
Nice utils, but given that they're only used in one pace, I might just leave them local to that file, until/unless they're needed elsewhere. I don't feel super strongly about this though if you think it adds value to extract them.